container.go

```diff
@@ -10,8 +14,9 @@ type Container struct {
 	position Position // The current position of the Container
 	Hidden   bool     // Is this Container hidden
 
-	Layout  Layout         // The Layout algorithm for arranging child CanvasObjects
-	Objects []CanvasObject // The set of CanvasObjects this container holds
+	Layout      Layout // The Layout algorithm for arranging child CanvasObjects
+	objectsLock *sync.Mutex
```

Comment:
Isn’t it possible to not use a pointer here and thus avoid the need to pass a new value each time the struct is created? The current way would crash anyone using containers without the constructors.

Reply:
```suggestion
	objectsLock sync.Mutex
```

Reply:
Definitely, no need for a pointer. Thanks